aplus/settings.py

```diff
@@ -104,6 +104,7 @@
     'django.contrib.auth',
     'django.contrib.humanize',
     'django.contrib.sitemaps',
+    'django.forms',
```

Comment:
The changes in settings.py were necessary to support widgets with custom templates.